src/browser/components/SettingsPage.jsx

```diff
@@ -29,6 +29,9 @@ import TabBar from './TabBar.jsx';
 const CONFIG_TYPE_SERVERS = 'servers';
 const CONFIG_TYPE_APP_OPTIONS = 'appOptions';
 
+// @eslint-ignore
+const {dialog} = require('electron').remote;
```

Comment:
Can this not just be declared as `remote.dialog` somewhere? We're already importing `remote` above.